### ISSUE TITLE
Syntax Error!

### ISSUE THREAD

**Issue by BaseMax** [mahdigosyan/mahdigosyan#1]
```
    currentFocus: "Backend Web Development",     
    funFact: "Bug's and Errors are my best friends"
    myIllness: "It may be strange for you to believe, but I am sick and crazy about {Books}"
```

The `,` token missed!

**Comment by BaseMax** [comment]
Fixed by #2 pull request.

**Comment by BaseMax** [comment]
Fixed by #2 pull request.
